src/pages/Configuracoes/ConferenciaLaboratorio/ConfigConferenciaLaboratorio.css

```
@@ -447,6 +417,159 @@ body.dark-mode .conf_lab_header.is-compact {
+}
+
+.conf_lab_hist_quando {
+    font-size: 0.78rem;
+    font-weight: 700;
+    color: #123e59;
+}
+
+.conf_lab_hist_arqs {
+    font-size: 0.72rem;
+    color: #3d5d72;
+    word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)